### ISSUE TITLE
Real Partial Mocks and Real Partial Relaxed Mocks

### ISSUE THREAD

**Issue by ATXMJ** [mockk/mockk#436]
Would it be possible for Mockk to support "real partial mocks" (as well as "real partial relaxed mocks", via a feature equivalent to [Mockito's `thenCallRealMethod()` feature](https://javadoc.io/doc/org.mockito/mockito-core/latest/org/mockito/Mockito.html#partial_mocks).

### Expected Behavior

You can create a mock that has all functions mocked **except** for a specified method, via a DSL "answer" syntax. I'd suggest something like:
```
val car = mockk<Car>(relaxed = true)
every { car.drive(any()) } calls realFunction
```

This would cause invocations of `car.drive()` to use the actual implementation.

### Current Behavior

Currently, to get this behavior, you'd need to use a `spyk<Car>()` object, and mock every function in the class **except** for the one you want to use the real implementation.

**Comment by oleksiyp** [comment]
Hi.

Have you tried `mockk(relaxed=true)` + `callOriginal` in `answers`?

**Comment by ATXMJ** [comment]
> Hi.
> 
> Have you tried `mockk(relaxed=true)` + `callOriginal` in `answers`?

Thank you! I think this is exactly what I was looking for! I guess this still needs to be added to the documentation?

**Comment by oleksiyp** [comment]
Yes, indeed, this is not described.

**Comment by ATXMJ** [comment]
@oleksiyp should we close this ticket or use it for documentation on the topic?

**Comment by oleksiyp** [comment]
Lets keep it

**Comment by Raibaz** [comment]
This is basically just adding proper documentation for the case described in the first issue and its solution.

**Comment by derlin** [comment]
Hi there ! Would be interested in taking this one. Would a section called "Relaxed Mock calling real method" below "Relaxed mock" be ok ?

**Comment by Raibaz** [comment]
Yep I think that'd be ok.

Thanks!

**Comment by Raibaz** [comment]
Closed by #710.
